### PR TITLE
[DOC] Move pages to Discover section

### PR DESCRIPTION
**Pull request by vagimeli**
Signed-off-by: vagimeli <vagimeli@amazon.com>

### Description
Move pages to Discover section; task related to doc site restructure initiative.

### Issues Resolved
#1406 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@Naarcha-AWS This is the PR to move the two pages to subpages under the Discover section. Do I have the correct labels attached to this PR? Also, please review the changes to the navigation text.

File: _dashboards/discover/dql.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
 layout: default
 title: Dashboards query language
-nav_order: 99
+parent: Discover
+nav_order: 1
```
Comment (Naarcha-AWS) [inline _dashboards/discover/dql.md:5]: We'll need to add a redirect from the previous page, use this syntax:

```
redirect_from: 
  - /dashboards/dql/
```

**Review by Naarcha-AWS — APPROVED** [review]

File: _dashboards/discover/dql.md (hunk 1 above)
Comment (vagimeli) [reply _dashboards/discover/dql.md:5]: @Naarcha-AWS Done. Thanks for the review.

**Review by JeffH-AWS — APPROVED** [review]

**Comment by vagimeli** [comment]
Duplicate. See #2621

**Pull request closed without merging**